ros/sonair_evk/sonair_evk/convert_coap_payload_to_pointcloud.py

```diff
@@ -1,4 +1,6 @@
-import struct
+# Copyright (C), 2024, Sonair AS
+#
+# This software is free software, licensed according to the MIT License:https://en.wikipedia.org/wiki/MIT_License import struct
 from typing import Tuple
```

Comment:
It is common to include a license file in the repo as well I think? See https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository